src/lib.rs

```diff
@@ -2303,17 +2304,18 @@ safety_comment! {
 }
 safety_comment! {
     /// SAFETY:
-    /// `Wrapping<T>` is guaranteed by its docs [1] to have the same layout as
-    /// `T`. Also, `Wrapping<T>` is `#[repr(transparent)]`, and has a single
-    /// field, which is `pub`. Per the reference [2], this means that the
-    /// `#[repr(transparent)]` attribute is "considered part of the public ABI".
+    /// `Wrapping<T>` is `#[repr(transparent)]` and has a single `T` field,
+    /// which is `pub`. [1] Per axiom-repr-transparent-layout-validity, we may
```

Comment:
Should `axiom-repr-transparent-layout-validity` perhaps be a intradoc link? It'd give us a modest defense against proof rot.

Reply:
Oooh good idea. I'll also need to make sure that our safety doc comments are actually being parsed by rustdoc.